.eslintignore

```diff
@@ -1,8 +1,5 @@
 # Ignore dist files
 client/dist/
 
-# Ignore legacy files
-client/src/legacy/
```

Comment:
It's not _actually_ legacy - we're not getting rid of it any time soon so it should meet our linting standards.